actors/miner/src/lib.rs

```diff
@@ -1484,6 +1495,16 @@ impl Actor {
         rt: &impl Runtime,
         sectors: Vec<SectorPreCommitInfoInner>,
     ) -> Result<(), ActorError> {
+        let miner_actor_id: u64 = if let Payload::ID(i) = rt.message().receiver().payload() {
```

Comment:
This is guaranteed to be an ID address, so just unwrap it.

Reply:
Done.